tsl/test/isolation/specs/compression_ddl.spec

```diff
@@ -20,16 +20,13 @@ teardown
 }
 
 session "I"
-step "Ib"   { BEGIN; SET LOCAL lock_timeout = '500ms'; SET LOCAL deadlock_timeout = '10ms';}
 step "I1"   { BEGIN; INSERT INTO ts_device_table VALUES (1, 1, 100, 100); }
 step "Ic"   { COMMIT; }
 
 session "S"
-step "St" {BEGIN;}
 step "S1" { SELECT count(*) from ts_device_table; }
 step "SC1" { SELECT count(*) from _timescaledb_internal._hyper_1_1_chunk; }
 step "SH" { SELECT total_chunks, number_compressed_chunks from hypertable_compression_stats('ts_device_table'); }
-step "Sc" {COMMIT;}
```

Comment:
If this is missing from the sequences below, is not that an error in itself?

Reply:
this PR only removes dead code, and an isolation test session is not required to be in a single transaction, to check progress in a session it's actually required not to run in transaction due to MVCC

Reply:
Well... yes, but why was the commit there to begin with if it was not needed?